types_test.go

```diff
@@ -9,18 +9,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestUnmarshalGeometryFromPointsArray(t *testing.T) {
```

Comment:
Please, don't remove the test. Just fix that. :)

Reply:
This test invalid - OSRM does not reply in format described in this test. In next commit I'm adding another test though.